core/trino-spi/src/main/java/io/trino/spi/eventlistener/OutputColumnMetadata.java

```diff
@@ -31,6 +32,7 @@
     private final Set<ColumnDetail> sourceColumns;
 
     @JsonCreator
+    @Unstable
```

Comment:
undo

Reply:
well, i added this here intentionally. Why would you want me to undo here?